generate_token.py

```
@@ -0,0 +1,42 @@
+from clerk_backend_api import Clerk
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)